src/view/event-panel-edit/editor-complete-view.jsx

```diff
@@ -42,6 +44,9 @@ import DatePicker from './components/date-picker';
 import RecurrenceSelector from './components/recurrence-selector';
 import DropZoneAttachment from './components/dropzone-component';
 
+const emailRegex =
```

Comment:
please use this regex instead, so that we can keep support for the Cyrillic language:

/[^\s@]+@[^\s@]+\.[^\s@]+/;